emf-builtins/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: emf-builtins
+Bundle-SymbolicName: io.lionweb.java.emf.builtins;singleton:=true
+Bundle-Version: 0.1.0.qualifier
```

Comment:
Is this the version of the Maven package? If so, we should consider generating this file

Reply:
Looking at the build.gradle file it seems generated, so I think we should not commit it, but adding the task generating this file as a dependency of the `setup` task